Add "Importing projects to VSCode" tab and Commonly used IDEs table (#512)
vaadin/docs: articles/guide/importing/vscode.asciidoc

---
title: Visual Studio Code
tab-title: VS Code
order: 400
layout: page
---

= Importing a Project in the Visual Studio Code

. In VS Code, select *"File > Open Folder..."*.
. Select the folder containing the Maven project you want to import.
+
image:images/vscode/open-folder.png[Open Folder]

. Click *Select Folder*.

> . A notification might appear asking you whether you would like to import Java projects contained in this workspace. Click "Yes" or "Always". 

Reviewer: ```suggestion
. A notification may appear asking you whether you would like to import Java projects contained in this workspace.
Click [guibutton]#Yes# or [guibutton]#Always#.
```
It doesn't always appear? It's a bit odd to say "may" in such case... Oh well.

Author: Yes, it doesn't appear if you don't have the java extension pack (or some specific java extension) installed.

+
image:images/vscode/import-java-projects.png[Open Folder]

> VS Code might suggest that you install additional extensions, here are the ones recommended for Vaadin:

Reviewer: ```suggestion
VS Code may suggest that you install additional extensions.
The following ones are recommended for Vaadin:
```
Again, "may" is a bit oddly vague.

Author: Well, if you have them installed already, it's not going to suggest them. It may suggest new ones, even though you installed all the recommended ones before.
